base/complex.jl

```diff
@@ -1,5 +1,13 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+"""
+    Complex{T<:Real}
```

Comment:
When documenting types I think it would be nice if it was standard to also include the supertype:
```
Complex{T<:Real} <: Number
```